src/makeRobot.js

```diff
@@ -37,8 +37,64 @@
  *
  * @return {Robot}
  */
+
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version}, wheels: ${this.wheels}`;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(n) {
+      if (n === undefined) {
+        this.coords.y++;
+      } else if (n >= 0) {
+        this.coords.y += n;
+      }
```

Comment:
```suggestion
    goForward(step = 1) {
      if (step >= 0) {
        this.coords.y += step;
      }

      return this;
    },
```

Виправ і для інших методів